modules/@apostrophecms/i18n/index.js

```diff
@@ -523,7 +523,15 @@ module.exports = {
               self.namespaces[ns].browser = self.namespaces[ns].browser ||
                 (metadata[ns] && metadata[ns].browser);
               const namespaceDir = path.join(localizationsDir, ns);
+              if (!fs.statSync(namespaceDir).isDirectory()) {
+                // Handle files in the namespace that aren't JSON files
```

Comment:
Currently, the conditional at the top of `for` loop checks if the current item is a JSON file and if it is skips it because it has been handled by the `addDefaultResourcesForModule()` method. At this point, the method is assuming that everything else is a directory, which isn't always true. Therefore we need to a check to jump to the next item in the loop if it is a file that doesn't have a `.json` extension. This would include `.DS_Store` that is breaking the build.
Maybe I'm not understanding your concern.